media-video/ffmpeg/ffmpeg-9999.ebuild

```diff
@@ -241,6 +241,7 @@ RDEPEND="
 	rubberband? ( >=media-libs/rubberband-1.8.1-r1[${MULTILIB_USEDEP}] )
 	samba? ( >=net-fs/samba-3.6.23-r1[client,${MULTILIB_USEDEP}] )
 	sdl? ( media-libs/libsdl2[sound,video,${MULTILIB_USEDEP}] )
+	sndio? ( media-sound/sndio:= )
```

Comment:
Why no `[${MULTILIB_USEDEP}]`?

Reply:
None, somehow forgot it.
Pushed a fixed version.